hbase-examples/src/main/java/org/apache/hadoop/hbase/client/example/HttpProxyExample.java

```diff
@@ -239,6 +240,7 @@ protected void initChannel(Channel ch) throws Exception {
         }).bind(port).syncUninterruptibly().channel();
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
This is example code so ideally we show how to properly manage futures here. In this case just adding a comment at the top of the file should be good, to explain the suppressions.

    /*
     * Future class methods will all return a new {@link Future}, so you always have one future
     * that will not been checked, so we need to suppress error-prone "FutureReturnValueIgnored"
     * warnings
     */